admin_guide/limits.adoc

```diff
@@ -93,20 +78,12 @@ spec:
 containers.
 <3> The maximum amount of memory that a pod can request on a node across all
 containers.
-<4> The minimum amount of CPU that a pod can request on a node across all
-containers. Not setting a `min` value or setting `0` is unlimited allowing the 
-pod to consume more than the `max` value.
-<5> The minimum amount of memory that a pod can request on a node across all
-containers. Not setting a `min` value or setting `0` is unlimited allowing the 
-pod to consume more than the `max` value. 
+<4> The minimum amount of CPU that a pod can request on a node across all containers. Not setting a `min` value or setting `0` is unlimited allowing the pod to consume more than the `max` value.
```

Comment:
s/is unlimited allowing/is unlimited, allowing (suggest this throughout this section)

Reply:
I strove for a change that is hopefully clearer.  Super fun that it's rubber stamped in a plurality of locations.

Reply:
It's minor but the sentence is still confusing to me. At the least, it could still use a comma to indicate a pause between "unlimited" and "allowing". But better yet, maybe something like, "There are no limits to setting a `min` value or setting `0`, both of which allow the pod to consume more than the `max` value." 🤷